cv32e40x/tests/programs/custom/debug_test_trigger/trigger_code.S

```diff
@@ -276,7 +278,7 @@ _wait3:
     # Enable debug_req in VP
     # ----------------------------------------------------
     li a1, debug_req_reg
-    li t0, 0xc005001a
+    li t0, 0xc005001c
```

Comment:
could these magic numbers be put into some constant like the previous CV_VP_* constants?

Reply:
This is actually an encoded VP register, but I will add macros to at least better document the test intention.